openwrt/configs/common.config

```diff
@@ -18,89 +18,53 @@ CONFIG_LUA_ECO_OPENSSL=y
 # CONFIG_LUA_ECO_WOLFSSL is not set
 CONFIG_OPENSSH_LIBFIDO2=y
 CONFIG_OPENSSL_ENGINE=y
+CONFIG_OPENSSL_OPTIMIZE_SPEED=y
 CONFIG_OPENSSL_PREFER_CHACHA_OVER_GCM=y
 CONFIG_OPENSSL_WITH_ASM=y
 CONFIG_OPENSSL_WITH_CHACHA_POLY1305=y
 CONFIG_OPENSSL_WITH_CMS=y
 CONFIG_OPENSSL_WITH_DEPRECATED=y
 CONFIG_OPENSSL_WITH_ERROR_MESSAGES=y
-CONFIG_OPENSSL_WITH_NPN=y
 CONFIG_OPENSSL_WITH_PSK=y
 CONFIG_OPENSSL_WITH_SRP=y
+CONFIG_OPENSSL_WITH_SSE2=y
 CONFIG_OPENSSL_WITH_TLS13=y
 CONFIG_PACKAGE_apinger=y
 CONFIG_PACKAGE_bash=y
 # CONFIG_PACKAGE_dnsmasq is not set
 # CONFIG_PACKAGE_dropbear is not set
 # CONFIG_PACKAGE_firewall4 is not set
 CONFIG_PACKAGE_iperf3=y
+# CONFIG_PACKAGE_jansson is not set
 CONFIG_PACKAGE_jq=y
-CONFIG_PACKAGE_kmod-ipt-core=y
-CONFIG_PACKAGE_kmod-nf-ipt=y
-CONFIG_PACKAGE_kmod-usb-serial=y
```

Comment:
Some of these kernel modules were for serial connectivity, but arent used since we just use the serial pins on the board instead of a dedicated usb device

Reply:
makes sense